src/main/java/frc/robot/subsystems/gpm/Shooter.java

```diff
@@ -36,12 +51,34 @@ public void periodic() {
         topMotor.set(topPID.calculate(topSpeed) + feedforward.calculate(topPID.getSetpoint()));
         bottomMotor.set(bottomPID.calculate(bottomSpeed) + feedforward.calculate(bottomPID.getSetpoint()));
 
-        SmartDashboard.putNumber("top speed", ConversionUtils.shooterRPMToSpeed(topSpeed));
-        SmartDashboard.putNumber("bottom speed", ConversionUtils.shooterRPMToSpeed(bottomSpeed));
+        SmartDashboard.putNumber("top speed", shooterRPMToSpeed(topSpeed));
+        SmartDashboard.putNumber("bottom speed", shooterRPMToSpeed(bottomSpeed));
         SmartDashboard.putBoolean("at setpoint?", atSetpoint());
-        System.out.println(ConversionUtils.shooterRPMToSpeed(topSpeed) + "," + ConversionUtils.shooterRPMToSpeed(bottomSpeed));
+        System.out.println(shooterRPMToSpeed(topSpeed) + "," + shooterRPMToSpeed(bottomSpeed));
     }
 
+	/**
+	* Converts an RPM to a speed at the surface of the shooter wheels.
+	*
+	* @param  rpm the RPM to convert
+	* @return     the converted value in m/s
+	* @see        shooterSpeedToRPM
+	*/
+	public static double shooterRPMToSpeed(double rpm) {
+			return (rpm / 60) * (4 * Math.PI * 0.0254);
```

Comment:
Wheel diameter should probably be  a constant, and you should use Units.inchessToMeters() instead of multiplying by 0.0254.